kubernetes_platform/proto/kubernetes_executor_config.proto

```diff
@@ -122,3 +124,22 @@ message PodMetadata {
     map<string, string> labels = 1;
     map<string, string> annotations = 2;
 }
+
+message ConfigMapAsVolume {
+    // Name of the ConfigMap.
+    string config_map_name = 1;
+    // Container path to mount the ConfigMap data.
+    string mount_path = 2;
+}
+
+message ConfigMapAsEnv {
+    // Name of the ConfigMap.
+    string config_map_name = 1;
+
+    message ConfigMapKeyToEnvMap {
+        // Corresponds to a key of the ConfigMap.
+        string config_map_key = 1;
+        // Env var to which configmap_key's data should be set.
+        string env_var = 2;
+    }

```

Comment:
Sorry @roytman, just found out your `ConfigMapKeyToEnvMap` definition wasn't defined as a list of map. can you also add
```
repeated ConfigMapKeyToEnvMap key_to_env = 2;
```

Thank you.

Reply:
sorry, fixed